src/es/components/molecules/tabs/Tabs.js

```diff
@@ -58,26 +58,26 @@ export default class Tabs extends Shadow() {
             content: '';
             width: 100%;
             height: 3px;
-            background-color: var(--m-orange-300);
+            background-color: var(--tab-navigation-background-color, var(--color-disabled));
```

Comment:
same here... --tab-navigation needed? if yes, use the same for all the elements within this css selector, which is going to make it easier to control in the future. or at least to guess the variable names.